apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/features/StaticLargeObjectApiLiveTest.java

```diff
@@ -88,11 +94,27 @@ public void testReplaceManifest() throws Exception {
    @Test(dependsOnMethods = "testReplaceManifest")
    public void testDelete() throws Exception {
       for (String regionId : regions) {
-         api.getStaticLargeObjectApi(regionId, containerName).delete(name);
+         DeleteStaticLargeObjectResponse resp = api.getStaticLargeObjectApi(regionId, containerName).delete(name);
+         assertThat(resp.status()).isEqualTo("200 OK");
+         assertThat(resp.deleted()).isEqualTo(3);
+         assertThat(resp.notFound()).isEqualTo(0);
+         assertThat(resp.errors()).isEmpty();
          assertEquals(api.getContainerApi(regionId).get(containerName).getObjectCount(), 0);
       }
    }
 
+   public void testDeleteSinglePartObjectWithMultiPartDelete() throws Exception {
+      String objectName = "testDeleteSinglePartObjectWithMultiPartDelete";
+      for (String regionId : regions) {
+         api.getObjectApi(regionId, containerName).put(objectName, newByteSourcePayload(ByteSource.wrap("swifty".getBytes())));
+         DeleteStaticLargeObjectResponse resp = api.getStaticLargeObjectApi(regionId, containerName).delete(objectName);
+         assertThat(resp.status()).isEqualTo("400 Bad Request");
```

Comment:
Ok, looks good.

But shouldn't the mock test for StaticLargeObjectApi also change to account for the response change?